src/reporter.ts

```diff
@@ -412,8 +410,14 @@ export class RPReporter implements Reporter {
       hasStats: false,
       startTime: this.client.helpers.now(),
     };
+
+    Object.defineProperty(step, 'id', {
+      value: randomUUID(),
+      configurable: true,
```

Comment:
Is it make sense to set this property as `configurable: false`?

Reply:
Okay, I will make it `configurable:false`. 
I left the ability to change the flags, because perhaps we may need it. But I wasn't sure if we needed it - so I'll remove it.